nautobot/extras/models/models.py

```diff
@@ -170,6 +172,7 @@ class ConfigContextModel(models.Model, ConfigContextSchemaValidationMixin):
         null=True,
         blank=True,
     )
+    # todoindex:
```

Comment:
```
nautobot/extras/datasources/git.py-733-        for record in model.objects.filter(
nautobot/extras/datasources/git.py-734-            local_context_data_owner_content_type=git_repository_content_type,
nautobot/extras/datasources/git.py:735:            local_context_data_owner_object_id=repository_record.pk,
nautobot/extras/datasources/git.py-736-        ):
nautobot/extras/datasources/git.py-737-            if record.name not in preserve_local[grouping]:
--
nautobot/extras/models/models.py-174-    )
nautobot/extras/models/models.py-175-    # todoindex:
nautobot/extras/models/models.py:176:    local_context_data_owner_object_id = models.UUIDField(default=None, null=True, blank=True)
nautobot/extras/models/models.py-177-    local_context_data_owner = GenericForeignKey(
nautobot/extras/models/models.py-178-        ct_field="local_context_data_owner_content_type",
nautobot/extras/models/models.py:179:        fk_field="local_context_data_owner_object_id",
nautobot/extras/models/models.py-180-    )
nautobot/extras/models/models.py-181-
```

Reply:
Instead of just adding an index to just `local_context_data_owner_object_id`, please index this together with `local_context_data_owner_content_type` via `Meta.indexes`.